### DISCUSSION TITLE
Platform Test ohne Erfolg

### DISCUSSION THREAD

**Discussion by Flip9Flip** [tbnobody/OpenDTU#973] [General]
Grüß Euch,

leider bekomme ich folgende Fehlermeldung, sobald ich den PlatformIO: Test fahre.

Building & Uploading...
Error: Nothing to build. Please put your test suites to the '/Users/Phil/OpenDTU/OpenDTU/test' folder
Uploading stage has failed, see errors above. Use `pio test -vvv` option to enable verbose output.

Eigentlich bin ich alles Schritt für Schritt durchgegangen. 

Vielen Dank vorab für die Unterstützung!

**Comment by tbnobody** [comment]
Warum möchtest du unbedingt einen Unit Test machen? Es sind keine implementiert...

**Reply by Flip9Flip** [reply]
Hab alles noch einmal neu (dieses mal mit einem Windows) geflasht. 
Leider bekomme ich die OpenDTU jedoch nicht im als WLAN angezeigt.

**Comment by tbnobody** [comment]
Was siehst du denn wenn du in die serielle Konsole schaust?

**Reply by Flip9Flip** [reply]
USB-SERIAL CH340 (COM8) 

Vor dem Flashen habe ich den "monitor_port" und "upload_port" auf COM8 gesetzt. 
Davor hatte ich ansonsten die Fehlermeldung (irgendwas mit fatal error com3 not found).
